code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/style/SmallImageUIStyle.kt

```diff
@@ -127,11 +128,8 @@ class SmallImageUIStyle private constructor(
             titleTextStyle = AepTextStyle.merge(defaultTitleAepTextStyle, titleAepTextStyle),
             bodyTextStyle = AepTextStyle.merge(defaultBodyAepTextStyle, bodyAepTextStyle),
             buttonRowStyle = AepRowStyle.merge(defaultButtonRowStyle, buttonRowStyle),
-            buttonStyle = (buttonStyle ?: arrayOfNulls(3)).map { pair ->
-                Pair(
-                    AepButtonStyle.merge(defaultButtonStyle, pair?.first),
-                    AepTextStyle.merge(defaultButtonTextStyle, pair?.second)
-                )
+            buttonStyle = (buttonStyle ?: arrayOfNulls(3)).map {
```

Comment:
nit: i believe buttonStyle is already an array of nulls or the specified overriden style at this point. i think the elvis operator can be removed to clean this up a little bit.